modules/nextflow/src/main/groovy/nextflow/k8s/model/PodHostMount.groovy

```diff
@@ -30,12 +30,29 @@ import groovy.transform.ToString
 @CompileStatic
 class PodHostMount {
 
+    private static Set<String> VALID_TYPES = [
+        'DirectoryOrCreate', 'Directory', 'FileOrCreate', 'File', 'Socket', 'CharDevice', 'BlockDevice'
+    ]
+
     String hostPath
 
     String mountPath
 
-    PodHostMount(String host, String container) {
+    boolean readOnly
+
+    String type // must be empty or a value from VALID_TYPES
```

Comment:
Is the type required by the readOnly capability or it's a separate extension?

Reply:
No it's not - I can remove those - initially I thought I would need them but then found that the readOnly was possible and much simpler. Let me know what you prefer.

Reply:
I'd suggest to remove then